### PR TITLE
Update Lecture3_BigCollaborations.md

### PR DESCRIPTION
**Pull request by thinderer**
Content mostly taken over from last year

### CODE REVIEW

File: Lectures/Lecture3_BigCollaborations.md [hunk 1]
```diff
@@ -1,41 +1,62 @@
...
-**Topics covered in lecture**
-+ topic 1
-+ topic 2
++ In preparation for this lecture, course participants are asked to watch the two-part video lecture by Dr Niels Martens on the epistemology of scientific collaborations. Lecture 1 of this set (https://www.youtube.com/watch?v=M1giIxeLuIg) discusses examples of 'big science' projects such as CERN and the Next Generation Event Horizon Telescope. Lecture 2 of this set (https://www.youtube.com/watch?v=DRt1uTytWf8) focuses on ways to manage the large number of co-authors that are involved in these projects and policies for giving appropriate credit. 
++ Course participants will work in small groups to investigate the credit structure of one collaboration they were assigned from the list below. They prepare a short presentation to their peers. 
++ The groups will give their presentations to the class and have a final critical discussion on the topic of large collaborations, for which Dr. Niels Martens will also join. 
 
 ## Evaluation
-__assignments or evaluation of the desired learning outcome__
++ Students answer the quiz about the video lectures by Dr. Niels Martens.
```
Comment (SanliFaez) [inline Lectures/Lecture3_BigCollaborations.md:31]: the quiz questions are about the podcast, not the videos.

File: Lectures/Lecture3_BigCollaborations.md [hunk 2]
```diff
@@ -1,41 +1,62 @@
...
++ Students give a short presentation of their findings to the class, receive peer feedback, and hear from the others on different examples.  
++ Students participate in the final discussion. 
 
 ### Assignment description
 
+Students work in small groups to investigate the credit structure of one of the large collaborations from the list below. They prepare a short presentation that addresses:
+
+1. A brief introduction to the collaboration, its scientific aims, when it was established, and its membership. 
+2. An assessment of how credit structures are implemented and how junior researchers can grow within this big collaboration and gain visibility in the field, as far as can be determined from the resources that are made publicly available by the collaboration. 
+
```
Comment (SanliFaez) [inline Lectures/Lecture3_BigCollaborations.md:42]: we also discussed to add an investigation of the open science skills and practices emphasized in the collaboration. Can we add that also as a third topic for the investigation?

**Review by SanliFaez — CHANGES_REQUESTED** [review]
This seems to me like a complete and fruitful exercise.
Will you not introduce the topic yourself @thinderer ? if yes, then i think your name should be in the lecturer field.

File: Lectures/Lecture3_BigCollaborations.md [hunk 3]
```diff
@@ -1,41 +1,62 @@
...
 
 ## Embedding/community
-__please mention the main community that should be considered when studying the topics of this lecture, e.g. individual researchers, research community, employers, general public, etc. __
+This lecture focuses on large national as well as international collaborations. Many of the large collaborations involve interdisciplinary components, both within physics and with engineering and industry. See https://bigscience.nl/about for a concerted effort in NL to foster synergies between physics and technology in large collaborations. 
 
 ## Methods/exercises
-**Topics covered in lecture**
-+ topic 1
-+ topic 2
++ In preparation for this lecture, course participants are asked to watch the two-part video lecture by Dr Niels Martens on the epistemology of scientific collaborations. Lecture 1 of this set (https://www.youtube.com/watch?v=M1giIxeLuIg) discusses examples of 'big science' projects such as CERN and the Next Generation Event Horizon Telescope. Lecture 2 of this set (https://www.youtube.com/watch?v=DRt1uTytWf8) focuses on ways to manage the large number of co-authors that are involved in these projects and policies for giving appropriate credit.
```
Comment (erikvansebille) [inline Lectures/Lecture3_BigCollaborations.md:26]: Is that true? Do they have to watch the video _and_ listen to the podcast? Or either?

File: Lectures/Lecture3_BigCollaborations.md [hunk 4]
```diff
@@ -1,41 +1,62 @@
...
++ Course participants will work in small groups to investigate the credit structure of one collaboration they were assigned from the list below. They prepare a short presentation to their peers. 
++ The groups will give their presentations to the class and have a final critical discussion on the topic of large collaborations, for which Dr. Niels Martens will also join. 
 
 ## Evaluation
-__assignments or evaluation of the desired learning outcome__
++ Students answer the quiz about the video lectures by Dr. Niels Martens. 
++ Students work in small groups to investigate an example of the credit structure and promotion of junior researchers within a large collaboration.
```
Comment (erikvansebille) [inline Lectures/Lecture3_BigCollaborations.md:32]: This turned out to be quite complex last year. Many collaborations weren't really explicit about their credit structure

File: Lectures/Lecture3_BigCollaborations.md [hunk 5]
```diff
@@ -1,41 +1,62 @@
...
+List of example collaborations to investigate:
+- Intergovernmental Panel on Climate Change  http://www.ipcc.ch
+- ALICE experiment at CERN  https://alice-collaboration.web.cern.ch
+- Einstein Telescope collaboration https://www.et-gw.eu/index.php/the-et-collaboration
+- Pangeo https://pangeo.io
+- Euclid consortium https://www.euclid-ec.org 
+- European Centre for Medium Range Weather Forecasts  https://www.ecmwf.int
+- Quantum Delta NL https://quantumdelta.nl
+- Cosmic-Ray Pierre Auger collaboration https://www.auger.org/collaboration 
+
```
Comment (erikvansebille) [inline Lectures/Lecture3_BigCollaborations.md:52]: Another option could be NESSC (https://www.nessc.nl)?

File: Lectures/Lecture3_BigCollaborations.md (hunk 1 above)
Comment (thinderer) [reply Lectures/Lecture3_BigCollaborations.md]: corrected this in the updated version

File: Lectures/Lecture3_BigCollaborations.md (hunk 3 above)
Comment (thinderer) [reply Lectures/Lecture3_BigCollaborations.md]: corrected, only the podcast and the video #1 will be shown in class

File: Lectures/Lecture3_BigCollaborations.md (hunk 4 above)
Comment (thinderer) [reply Lectures/Lecture3_BigCollaborations.md]: added that in that case, they can look up some papers by the collaboration and at least say if there are, for example, multiple tiers of authors. That won't cover the full credit structure but at least gives more indication than no information.

File: Lectures/Lecture3_BigCollaborations.md (hunk 5 above)
Comment (thinderer) [reply Lectures/Lecture3_BigCollaborations.md]: good, added this

File: Lectures/Lecture3_BigCollaborations.md [hunk 6]
```diff
@@ -1,41 +1,62 @@
...
-**Topics covered in lecture**
-+ topic 1
-+ topic 2
++ In preparation for this lecture, course participants are asked to listen to the podcast and answer the quiz questions
++ In lecture, we watch the first of the two-part video lecture series by Dr Niels Martens on the epistemology of scientific collaborations. [Lecture 1 of this set](https://www.youtube.com/watch?v=M1giIxeLuIg), which is covered in class, discusses examples of 'big science' projects such as CERN and the Next Generation Event Horizon Telescope. [Lecture 2](https://www.youtube.com/watch?v=DRt1uTytWf8), which is left for students to watch on their own, focuses on ways to manage the large number of co-authors that are involved in these projects and policies for giving appropriate credit. 
++ Course participants work in small groups on the assignment described below, where they investigate the implementation of open science principles and credit structure of a large collaboration. They prepare a short presentation to their peers. 
++ The groups present their findings to the class and participate in a final discussion on the topic of large collaborations, for which Dr. Niels Martens will also join.
```
Comment (erikvansebille) [inline Lectures/Lecture3_BigCollaborations.md:29]:
```suggestion
+ Course participants work in small teams on the assignment described below, where they investigate the implementation of open science principles and credit structure of a large collaboration. They prepare a short presentation to their peers. 
+ The teams present their findings to the class and participate in a final discussion on the topic of large collaborations, for which Dr. Niels Martens will also join. 
```
I thought we decided to use the term 'groups' for the three big groups (each in their own tutorial room) and 'teams' for the smaller subsets of 5-6 students?

File: Lectures/Lecture3_BigCollaborations.md [hunk 7]
```diff
@@ -1,41 +1,62 @@
...
+This lecture focuses on large national as well as international collaborations. Many of the large collaborations involve interdisciplinary components, both within physics and with engineering and industry. See https://bigscience.nl/about for a concerted effort in NL to foster synergies between physics and technology in large collaborations. 
 
 ## Methods/exercises
-**Topics covered in lecture**
-+ topic 1
-+ topic 2
++ In preparation for this lecture, course participants are asked to listen to the podcast and answer the quiz questions
++ In lecture, we watch the first of the two-part video lecture series by Dr Niels Martens on the epistemology of scientific collaborations. [Lecture 1 of this set](https://www.youtube.com/watch?v=M1giIxeLuIg), which is covered in class, discusses examples of 'big science' projects such as CERN and the Next Generation Event Horizon Telescope. [Lecture 2](https://www.youtube.com/watch?v=DRt1uTytWf8), which is left for students to watch on their own, focuses on ways to manage the large number of co-authors that are involved in these projects and policies for giving appropriate credit.
```
Comment (erikvansebille) [inline Lectures/Lecture3_BigCollaborations.md:27]: In the lecture (i.e. the one big room)? Or during the tutorial (i.e. the three smaller rooms)?

File: Lectures/Lecture3_BigCollaborations.md [hunk 8]
```diff
@@ -1,41 +1,62 @@
...
++ Course participants work in small groups on the assignment described below, where they investigate the implementation of open science principles and credit structure of a large collaboration. They prepare a short presentation to their peers. 
++ The groups present their findings to the class and participate in a final discussion on the topic of large collaborations, for which Dr. Niels Martens will also join. 
 
 ## Evaluation
-__assignments or evaluation of the desired learning outcome__
++ Students answer the quiz about the podcast and participate in the discussion of the material covered. 
++ In class, students see the video lecture on big collaborations by Dr. Niels Martens and participate in the discussion of the material.
```
Comment (erikvansebille) [inline Lectures/Lecture3_BigCollaborations.md:33]: Is 'class' here the lecture or the tutorial?

File: Lectures/Lecture3_BigCollaborations.md [hunk 9]
```diff
@@ -1,41 +1,62 @@
...
++ Course participants work in small groups on the assignment described below, where they investigate the implementation of open science principles and credit structure of a large collaboration. They prepare a short presentation to their peers. 
++ The groups present their findings to the class and participate in a final discussion on the topic of large collaborations, for which Dr. Niels Martens will also join. 
 
 ## Evaluation
-__assignments or evaluation of the desired learning outcome__
++ Students answer the quiz about the podcast and participate in the discussion of the material covered. 
++ In class, students see the video lecture on big collaborations by Dr. Niels Martens and participate in the discussion of the material. 
++ Students work in small groups to investigate an example of the credit structure, promotion of junior researchers, and open science principles within an example of a large collaboration.
++ Students give a short presentation of their findings to the class, receive peer feedback, and hear from the others on different examples.
```
Comment (erikvansebille) [inline Lectures/Lecture3_BigCollaborations.md:35]:
```suggestion
+ Students give a short presentation of their findings to the group, receive peer feedback, and hear from the others on different examples.  
```

File: Lectures/Lecture3_BigCollaborations.md [hunk 10]
```diff
@@ -1,41 +1,62 @@
...
++ Course participants work in small groups on the assignment described below, where they investigate the implementation of open science principles and credit structure of a large collaboration. They prepare a short presentation to their peers. 
++ The groups present their findings to the class and participate in a final discussion on the topic of large collaborations, for which Dr. Niels Martens will also join. 
 
 ## Evaluation
-__assignments or evaluation of the desired learning outcome__
++ Students answer the quiz about the podcast and participate in the discussion of the material covered. 
++ In class, students see the video lecture on big collaborations by Dr. Niels Martens and participate in the discussion of the material. 
++ Students work in small groups to investigate an example of the credit structure, promotion of junior researchers, and open science principles within an example of a large collaboration.
```
Comment (erikvansebille) [inline Lectures/Lecture3_BigCollaborations.md:34]:
```suggestion
+ Students work in small teams to investigate an example of the credit structure, promotion of junior researchers, and open science principles within an example of a large collaboration.
```

File: Lectures/Lecture3_BigCollaborations.md [hunk 11]
```diff
@@ -1,41 +1,62 @@
...
++ Students give a short presentation of their findings to the class, receive peer feedback, and hear from the others on different examples.  
 
 ### Assignment description
 
+Students work in small groups to investigate the credit structure of one of the large collaborations from the list below, which they will be assigned. They prepare a short presentation using the [template](https://github.com/SanliFaez/OS4Physicists/blob/thinderer-patch-1/Assignments/BigCollaboration_presentation_template.md). The investigation and presentation covers the following points:
```
Comment (erikvansebille) [inline Lectures/Lecture3_BigCollaborations.md:39]:
```suggestion
Students work in small teams to investigate the credit structure of one of the large collaborations from the list below, which they will be assigned. They prepare a short presentation using the [template](https://github.com/SanliFaez/OS4Physicists/blob/thinderer-patch-1/Assignments/BigCollaboration_presentation_template.md). The investigation and presentation covers the following points:
```

File: Lectures/Lecture3_BigCollaborations.md (hunk 7 above)
Comment (thinderer) [reply Lectures/Lecture3_BigCollaborations.md]: corrected

File: Lectures/Lecture3_BigCollaborations.md (hunk 8 above)
Comment (thinderer) [reply Lectures/Lecture3_BigCollaborations.md]: corrected

**Review by SanliFaez — APPROVED** [review]

**Pull request merged**